spec/integration/flags_spec.rb

```diff
@@ -2,15 +2,15 @@
 
 RSpec.describe PostAction do
   it "triggers the 'flag_reviewed' event when there was at least one flag" do
-    admin = Fabricate(:admin, refresh_auto_groups: true)
```

Comment:
There was nothing significant about this user being an admin. They just need to be in the right TL group.